dgraphtest/acl.go

```diff
@@ -0,0 +1,428 @@
+/*
+ * Copyright 2023 Dgraph Labs, Inc. and Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dgraphtest
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/dgraph-io/dgraph/testutil"
+	"github.com/pkg/errors"
+)
+
+var (
+	errGroupCountMoreThanOne = errors.New("group count is other than 1")
```

Comment:
this could be more precise, I feel like "group count" is a generic term. Maybe add ACL to the message and the variable name.